Dockerfile

```diff
@@ -25,7 +26,7 @@ RUN apt-get -o Acquire::Check-Valid-Until=false update && \
 
 RUN git clone --depth 1 https://github.com/neighborhoods/tc_aws.git /tc_aws
 COPY requirements.txt requirements.txt
-RUN pip install --upgrade pip==9.0.3 && pip install --no-cache-dir -r requirements.txt
+RUN pip install --upgrade pip==20.0.2 && pip install --no-cache-dir -r requirements.txt
```

Comment:
why pin the version of pip? Why not just leave it at `pip install --upgrade pip`?

Reply:
It's preferable to pin dependency versions where its easy to since it makes it less likely later builds of the same image will fail.